outlook/src/taskpane/components/Helpdesk/TicketList/TicketListItem.tsx

```diff
@@ -3,41 +3,49 @@ import HelpdeskTicket from "../../../../classes/HelpdeskTicket";
 import '../../../../utils/ListItem.css'
 import { _t } from "../../../../utils/Translator";
 import api from "../../../api";
+import AppContext from "../../AppContext";
 import Logger from "../../Log/Logger";
 
 type TicketListItemProps = {
     ticket: HelpdeskTicket
 };
 
 
-const TicketListItem = (props: TicketListItemProps) => {
+class TicketListItem extends React.Component<TicketListItemProps, {}>{
 
-    const openInOdoo = () => {
-        let url = api.baseURL+`/web#id=${props.ticket.id}&model=helpdesk.ticket&view_type=form`;
+    openInOdoo = () => {
+        const  cids = this.context.getUserCompaniesString();
+        let url = api.baseURL+`/web#id=${this.props.ticket.id}&model=helpdesk.ticket&view_type=form${cids}`;
         window.open(url,"_blank");
     }
 
-    let closedText = null;
-
-    if (props.ticket.isClosed)
-    {
-        closedText = (<div className="muted-text" style={{marginTop: "8px", fontSize: "14px"}}>{_t("Closed")}</div>)
-    }
-
-    return (
-        <div className="list-item-root-container" onClick={openInOdoo}>
-            <div className="list-item-container">
-                <div className="list-item-info-container">
-                    <div className="list-item-title-text">
-                        {props.ticket.name}
+    render() {
+        let closedText = null;
+
+        if (this.props.ticket.isClosed)
```

Comment:
You know what Awa says with brackets :D

Reply:
Yes :D these old PR's were done without a linter.

It has now been corrected!